website/pages/use-cases/simple-container-orchestration/content.json

```diff
@@ -0,0 +1,153 @@
+{
+  "features": [
+    {
+      "textSplit": {
+        "heading": "Container Orchestration",
+        "content": "Deploy, manage, and scale your containers with the Docker, Podman, or Singularity task driver.",
+        "textSide": "right",
+        "links": [
+          {
+            "text": "Read More",
+            "url": "https://learn.hashicorp.com/collections/nomad/manage-jobs",
+            "type": "outbound"
+          }
+        ]
+      },
+      "codeBlock": {
+        "options": { "showWindowBar": true },
+        "language": "hcl",
+        "code": "task \"webservice\" {\n  driver = \"docker\"\n  config {\n    image = \"redis:3.2\"\n    labels {\n      group = \"webservice-cache\"\n    }\n  }\n}\n"
+      }
+    },
+    {
+      "textSplit": {
+        "heading": "Run On-Premise with Ease",
+        "content": "Install and run Nomad easily on bare metal as a single binary and with the same ease as on cloud."
+      },
+      "image": {
+        "url": "https://www.datocms-assets.com/2885/1608677608-on-prem-with-ease.svg",
```

Comment:
I've uploaded some of these assets to Dato, rather than having them in files local to the project. Main intention here was to make content a bit easier to separate from components being used. Happy to re-adjust / add back local files if we feel that'd be preferable.